heimdall2/templates/heimdall-statefulset.yaml

```diff
@@ -64,46 +64,167 @@ spec:
                 secretKeyRef:
                   name: {{ include "heimdall.fullname" . }}
                   key: databasePassword
+            - name: DATABASE_SSL
+              value: {{ .Values.databaseSsl | quote }}
+            - name: DATABASE_SSL_INSECURE
+              value: {{ .Values.databaseSslInsecure | quote }}
+            - name: DATABASE_SSL_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "heimdall.fullname" . }}
+                  key: databaseSslKey
+            - name: DATABASE_SSL_CERT
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "heimdall.fullname" . }}
```

Comment:
What does this do?

Reply:
Talked about this over slack.

The helper template defines some values like heimdall.fullname which we can use in various places safely (ex. identifiers are sometimes capped at 63 chars).